cmd/kube-scheduler/app/server_test.go

```diff
@@ -249,6 +249,18 @@ profiles:
 				t.Fatal(err)
 			}
 
+			nfs := opts.Flags()
```

Comment:
Was the move necessary?

Reply:
For this particular test, maybe not as it may not test any featuregate.

Howeve, given that now `options.NewOptions()` doesn't return a fully-instantiated CC, I'd prefer to always move flag parsing prior to `options.Complete()`; otherwise users may copy/paste this snippet improperly in the future.